### DISCUSSION TITLE
Admin User login failure & How to activate the FilamentUserResources in v1.8.1

### DISCUSSION THREAD

**Discussion by 3pleo** [chiiya/filament-access-control#41] [Q&A] [answered]
Hi,
I have been struggling to get Filament-access-control v1.8.1 up on new Filament installation.

I have a new Filament installation v2.17.50 on Laravel 10.14.1.
Looks like the filament-access-control documentation outdated or incomplete. After following the installation instructions, including updating config/filament.php with the Chiiya\…\Login, yet

1. Cannot login with users created by “filament-access-control:user”. FilamentAccessControl Seems to default to using the App\Models\User instead of the Chiiya\FilamentAccessControl\Models\FilamentUser. I had to manually alter config/auth.php to change the User Provider to FilamentUser before I can login. Is this a missing step in the documentation. Or I’m I missing something.
2. Also, I cannot figure out how to enable the FilamentAccessControl User Administration Resources. I have good experience using Filament & Laravel, but I have not been able to get this plugin up and running for over 24 hours by following the guide.

Kindly help.

Regards,
3pleO

**Comment by chiiya** [comment, marked as answer]
Did you adjust the `filament.auth.guard` config like it says in the README? Sounds to me like it's still using the `web` guard for you.

**Comment by 3pleo** [comment]
Thanks.
I changed the filament.auth.guard from “web” to “filament”, and it works!

Thanks.
